bylaws/officers.md

```diff
@@ -1,18 +1,54 @@
 # Officers of the Corporation
 ## Offices held
-The officers of this corporation shall be a Secretary Functionary, a Treasurer Functionary, and at least one Facilitation Functionary.
+The officers of this corporation shall be a number of Functionaries, with a minimum of a Secretary Functionary and Treasurer Functionary at all times. The corporation, at the board's discretion, may also have a chairman of the board.
 
-Any number of offices may be held by the same person, except that neither the Secretary Functionary nor the Treasurer Functionary may serve concurrently as either the President or Chairman of the Board.
+Neither the Secretary Functionary nor the Treasurer Functionary may serve concurrently as either the President or Chairman of the Board.
 
 {% include "../bylaw-pending.md"%}
-## Responsibilities
+
+## Appointment of the Chairman of the Board
+At the board's discretion, a member of the board of directors may be may be chosen annually by the board to be appointed to the office of chairman of the board and shall serve at the pleasure of the board, subject to the rights of any officer under any employment contract.
```

Comment:
No, they can change whenever. Per "vacancies in office":

> A vacancy in any office because of death, resignation, removal, disqualification, or any other cause shall be filled in the manner prescribed in these bylaws for normal assumption to that office, provided, however, that vacancies need not be filled on an annual basis.

The annual clause you point out only stipulates that the board is automatically allowed to replace the chairperson every year, and it is "at the pleasure of the board" meaning they can be "fired" at-will. The vacancies section says if chairperson is gone, the board can appoint someone else following the same procedure of selection.